src/view/MenuView.java

```diff
@@ -22,7 +25,8 @@ public class MenuView extends JPanel implements ActionListener, PropertyChangeLi
     public final JButton loadGame;
     public final JButton careerStats;
 
-    public MenuView(MenuViewModel menuViewModel, ViewManagerModel viewManagerModel) {
+    public MenuView(MenuViewModel menuViewModel, ViewManagerModel viewManagerModel,
+                    final StatisticsViewModel statisticsViewModel, final StatisticsController statisticsController) {
```

Comment:
I'm not really sure whats going on here. You are using the `final` keyword in the actual function header, and not storing the `StatisticsViewModel` or `StatisticsController` locally in `MenuView`, they only exist in your constructor. This will need to be changed to address my other comment in this file about CA best practice.